File: knap/knap.cs

using System;
public class Solution
{
	public  double knapsack(int []arr1,int []arr2,int n,int W)
        {
        
			double maxi_value = 4.56;
            
	//Write your code here without removing the exising code
	//the variable 'arr1' and 'arr2' contains array of integers of size n.
	//modified the double maxi_value contain the output of the program upto 2 place of decimal.

		    return maxi_value;
		   
           
        }
         public static void Main(string[] args)
        {
	        int n,W;
            n =Int32.Parse(Console.ReadLine());
            
            int[] arr1 = new int[n];
            int[] arr2 = new int[n];
            string[] s1 = Console.ReadLine().Split(' ');
            for (int i=0;i< n;i++)
            {
                arr1[i] = Int32.Parse(s1[i]);
            }
            string[] s2 = Console.ReadLine().Split(' ');
            for (int i=0;i<n;i++)
            {
                arr2[i] = Int32.Parse(s2[i]);
            }
            W=Int32.Parse(Console.ReadLine());
            Solution g= new Solution();
            
            double ans=g.knapsack(arr1,arr2,n,W);
             Console.WriteLine(ans);
            
           
        }
}

File: knap/knap.java

	
	public static void main(String[] args)
    {
		Scanner sc=new Scanner(System.in);
        int n = sc.nextInt(); 
        int []arr1= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
        int []arr2= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
        Solution g=new Solution();
        int W = sc.nextInt(); 
        double ans=g.knapsack(arr1,arr2,n,W);
        System.out.println(String.format("%.2f",ans));
	}
}
class Solution
{
	public static double knapsack(int weights[],int values[],int n,int W)
        {
        
			double maxi_value = 12.34;
        //Write your code here without removing the existing code.
	//the variable 'weights' and 'values' contains array of integers of size n.
	//modified the double maxi_value containing the output of the program up to 2 places of decimal.
		    return maxi_value;
        }
}